CMakeLists.txt

```diff
@@ -658,6 +658,8 @@ endif()
 
 if(MSVC)
     add_compile_definitions(_CRT_SECURE_NO_WARNINGS)
+    # add_compile_options(/utf-8)
```

Comment:
Remove commented code.

You said `add_compile_options` "didn't work as expected". What happened instead?